docs/docsource/installation/install/installing_xdm.rst

```diff
@@ -0,0 +1,517 @@
+.. _Installing XDM:
+
+Installing XDM
+==============
+
+.. note::
+   
+   This installation procedure is performed on a single node.
+
+Prerequisites
+-------------
+
+- Your disk must be partioned as follows:
+   
+  +-------------+--------+-----------+
+  | Partition # |  Size  |  Service  |
+  +=============+========+===========+
+  | 1           | 300GiB | Mongo     |
+  +-------------+--------+-----------+
+  | 2           | 100GiB | Kafka     |
+  +-------------+--------+-----------+
+  | 3           | 100GiB | S3Data    |
+  +-------------+--------+-----------+
+  | 4           | 10GiB  | Redis     |
+  +-------------+--------+-----------+
+  | 5           | 10GiB  | Zookeeper |
+  +-------------+--------+-----------+
+
+  If necessary, partition your disk with:
+
+  .. code::
+      
+     DISK_NAME=/dev/vdb
+     parted -a none ${DISK_NAME} --script \
+         mklabel gpt \
+         mkpart primary ext4 1MiB 320GiB \
+         mkpart primary ext4 320GiB 420GiB \
+         mkpart primary ext4 420GiB 520GiB \
+         mkpart primary ext4 520GiB 530GiB \
+         mkpart primary ext4 530GiB 540GiB \
+         mkpart primary ext4 540GiB 550GiB \
+         mkpart primary ext4 550GiB 560GiB \
+         mkpart primary ext4 560GiB 570GiB \
+         mkpart primary ext4 570GiB 580GiB
+
+Deploying MetalK8s
+------------------
+
+For more information on MetalK8s installation, see *Installation* in the MetalK8s' documentation.
+
+Deploying XDM-Operator
+----------------------
+
+#. From your terminal, create a namespace for XDM with:
```

Comment:
too many  "with:"
From the terminal, create a namespace for XDM. Simpler and better :)